test/integration/state/initialState.t.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity 0.8.23;
+
+import { IntegrationBaseSetup, IPowerToken, IZeroToken } from "../IntegrationBaseSetup.t.sol";
+
+contract StandardGovernorSetKey_IntegrationTest is IntegrationBaseSetup {
+    function test_initialState() external {
```

Comment:
```suggestion
    function test_initialState() external view {
```

Reply:
It doesn't compile otherwise if we keep view.

Reply:
Odd